tools/koreai/src/main/java/org/finos/symphony/toolkit/koreai/request/KoreAIRequesterImpl.java

```diff
@@ -34,26 +38,37 @@
     protected JsonNodeFactory jnf;
     protected String jwt;
     protected String url;
+    protected TrustManagerFactory tmf;
```

Comment:
How has this ended up here?  It doesn't seem to be related to your change

Reply:
I have not done any changes. Even git history also not showing any change on my name